src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -491,4 +498,26 @@ public static String replaceTokens(Map<String, ?> tokens, String value) {
 
         return buf.toString();
     }
+
+    public static void setupIDEResourceCopy(@Nonnull final Project project) {
+        if (project.getPlugins().hasPlugin(IdeaPlugin.class)) {
+            final IdeaPlugin idea = project.getPlugins().getPlugin(IdeaPlugin.class);
+            project.getTasks().create(CopyIDEAResources.NAME, CopyIDEAResources.class, task -> task.configure(idea.getModel()));
+        } else {
+            project.getLogger().warn("'idea' plugin is not found! IDEA resources will not be copied.");
```

Comment:
I'm not fond of logging warning messages when either or both of the plugins are not present. Could we instead have it log a warning when neither of the plugins are found?